api/src/main/java/org/openmrs/CohortMember.java

```diff
@@ -61,19 +66,19 @@ public void setCohortMemberId(Integer cohortMemberId) {
         this.cohortMemberId = cohortMemberId;
     }
 
-	public Integer getCohortId() {
+	public Cohort getCohortId() {
```

Comment:
Should I deprecate, even though I created those methods after I refactored based on your feedback? Those won't be used anywhere.